src/ui/components/Library/Team/utils.ts

```diff
@@ -47,3 +47,10 @@ export function pushRoute(router: NextRouter, relativeURL: string): void {
     search,
   });
 }
+
+export function useGetRealTeamId() {
+  const teamId = useGetTeamIdFromRoute();
+  const match = atob(teamId).match(/w:(.*)/);
+
+  return match ? match[1] : teamId;
```

Comment:
This feels silly but necessary. @bvaughn are you aware of somewhere where we already do this, in case there's some better established pattern?